macros/sql/get_query_results_as_single_value.sql

```diff
@@ -0,0 +1,29 @@
+{% macro get_query_results_as_single_value(query, row_position=0, column_position=0) %}
+    {{ return(adapter.dispatch('get_query_results_as_single_value', 'dbt_utils')(query, row_position, column_position)) }}
+{% endmacro %}
+
+{% macro default__get_query_results_as_single_value(query, row_position=0, column_position=0) %}
+
+{# This macro returns the (row_position, column_position) record in a query as a string #}
+    {%- set nth_row = row_position -%}
+    {%- set nth_column = column_position -%}
+
+    {%- call statement('get_query_result', fetch_result=True,auto_begin=false) -%}
+
+        {{ query }}
+
+    {%- endcall -%}
+
+    {%- if execute -%}
+
+        {% set sql_result = load_result('get_query_result').table.columns[nth_column].values()[nth_row]|string %}
```

Comment:
```suggestion
        {% set sql_result = load_result('get_query_result').table.columns[nth_column].values()[nth_row] %}
```
For the same reason as the note below, I don't think we want to cast this?